CONTRIBUTING.md

```diff
@@ -1,9 +1,26 @@
 # Contributing to Rust
 
-Thank you for your interest in contributing to Rust!
+Thank you for your interest in contributing to Rust! There are many ways to contribute
+and we appreciate all of them.
 
 To get started, read the [Contributing to Rust] chapter of the [rustc-dev-guide].
```

Comment:
Instead of inlining part of the rustc-dev-guide, can we just rephrase this sentence to make it friendlier to new contributors? E.g., I'm thinking something like:

```suggestion
Documentation for contributing to Rust is located in the [Guide to Rustc Development],
commonly known as the rustc-dev-guide. Despite the name, this guide documents
not just how to develop rustc (the Rust compiler), but also how to contribute to any part
of the Rust project.

To get started with contributing, please read the [Contributing to Rust] chapter of the guide.
That chapter explains how to get your development environment set up and how to get help.
```

Reply:
I was actually suggesting that you remove the other changes in the PR and *only* make the first sentence friendlier (of course, you don't have to use my exact phrasing for the revised sentence, I was just giving a general idea of what I meant).

The other information is covered in the dev-guide pretty early on already IIRC.